docs/riskAuthentication.md

```diff
@@ -58,25 +58,27 @@ If the `Device Profiling Method` setting in your adapter configuration is set to
 
 ### Setup
 
-If the `Device Profiling Method` setting in your adapter configuration is set to `Captured by a previous adapter` and `Device Profiling Script Source` is set to `ThreatMetrix SDK`:
+If the `Device Profiling Method` setting in your adapter configuration is set to `Use existing ThreatMetrix session ID` and `Device Profiling Script Source` is set to `ThreatMetrix SDK`:
```

Comment:
Wasn't this renamed to just `Device Profiling`?

Reply:
So it was. Will update.